tests/openwisp2/tests.py

```diff
@@ -1,14 +0,0 @@
-from openwisp_controller.tests.test_users_integration import (
-    TestUsersIntegration as BaseTestUsersIntegration,
```

Comment:
I think the same discussion applies here:
https://github.com/openwisp/openwisp-ipam/pull/86#discussion_r601511384

If I remember correctly, `UsersIntegrations` tests are still required.
Although, If you think I am incorrect here, please let me know, It's been some time since I've read this code, so I'll read the Integration tests and ping you back! :smile:

Reply:
P.S: Conversation#2: https://github.com/openwisp/openwisp-ipam/pull/86#discussion_r601558912

Reply:
P.P.S: Could you cross-check with openwisp-radius as well? 😄

Reply:
> Although, If you think I am incorrect here, please let me know, It's been some time since I've read this code, so I'll read the Integration tests and ping you back!

I am still confused about this. :confused:

Reply:
P.S.:- https://github.com/openwisp/openwisp-ipam/pull/86#discussion_r602743483

Reply:
> P.P.S: Could you cross-check with openwisp-radius as well?

I checked openwisp-radius and it is included there too

Reply:
P.S.:- I may checked wrong repository that time. `openwisp-radius` has no such tests

Reply:
Great, checking the rest then...